addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -171,14 +174,13 @@ export async function start(param0 = {}) {
     });
     param0["target"] = target;
     const pyEnv = await getPyEnv();
-    patchWithCleanup(sessionInfo, {
-        user_context: {
-            ...sessionInfo.user_context,
-            uid: pyEnv.currentUserId,
-        },
-        uid: pyEnv.currentUserId,
+    patchUserWithCleanup({
+        userId: pyEnv.currentUserId,
         name: pyEnv.currentUser?.name,
-        partner_id: pyEnv.currentPartnerId,
+        partnerId: pyEnv.currentPartnerId,
+    });
+    patchUserContextWithCleanup({
+        uid: pyEnv.currentUserId,
```

Comment:
This is a bit worrisome, isn't/shouldn't user context uid automatically synchronized with actual user id?

Reply:
Good point. Actually, I think it, I'll remove this patch and see what happens